crates/ra_assists/Cargo.toml

```diff
@@ -12,13 +12,13 @@ rustc-hash = "1.1.0"
 itertools = "0.9.0"
 either = "1.5.3"
 
-stdx = { path = "../stdx" }
+stdx = { path = "../stdx", version = "0.1.0", package = "ra_stdx" }
 
-ra_syntax = { path = "../ra_syntax" }
-ra_text_edit = { path = "../ra_text_edit" }
-ra_fmt = { path = "../ra_fmt" }
-ra_prof = { path = "../ra_prof" }
-ra_db = { path = "../ra_db" }
-ra_ide_db = { path = "../ra_ide_db" }
-hir = { path = "../ra_hir", package = "ra_hir" }
-test_utils = { path = "../test_utils" }
+ra_syntax = { path = "../ra_syntax", version = "0.1.0" }
```

Comment:
Is it possible to avoid specifying the versions in Cargo.toml?

Reply:
I don't think so. It is required for publishing to crates.io